### ISSUE TITLE
Support for the AMR-WB codec for HD Voice on T-Mobile

### ISSUE THREAD

**Issue by nickderobertis** [livekit/sip#94]
Hello, thanks for your great work on this sytem.

I am wondering if we can add AMR-WB codec support. This seems to be the codec used by T-mobile for HD voice. I've been unable to get HD voice working on any call yet but my phone is T-mobile. 

I saw the list of supported codecs [here ](https://github.com/livekit/sip/blob/418097f21eff8b87531004b428a25933ccbd4a0f/pkg/sip/media_codecs.go#L18-L22) (G722, G711U) so that's why I assumed AMR-WB has not been implemented yet.

**Comment by dennwc** [comment]
That's correct, for now we plan to fix G722 first, then add OPUS and probably G729. I think once we get G722 working correctly, adding new codecs would be pretty simple. But no ETA for now.

**Comment by ThundR67** [comment]
Any updates on this?
